frontend/src/views/WheresMyGene/components/HeatMap/style.ts

```diff
@@ -82,17 +80,3 @@ export const ChartWrapper = styled.div`
   left: ${Y_AXIS_CHART_WIDTH_PX}px;
   top: ${X_AXIS_CHART_HEIGHT_PX}px;
 `;
-
-/**
- * This is needed so that the heatmap scrollbar isn't covered by the right sidebar
- */
-function isFmgOpen({ isFmgOpen = false }: { isFmgOpen: boolean }) {
```

Comment:
Removing this because it introduced a bug where PNGs were getting cut off

Reply:
I don't have time to test this - but I'm assuming the heatmap scrollbar won't be covered by the right sidebar after removing this?